tests/Pagination/CursorPaginatorTest.php

```diff
@@ -101,7 +101,7 @@ public function testReturnEmptyCursorWhenItemsAreEmpty()
     {
         $cursor = new Cursor(['id' => 25], true);
 
-        $p = new CursorPaginator(Collection::make(), 25, $cursor, [
+        $p = new CursorPaginator(new Collection(), 25, $cursor, [
```

Comment:
drop the parentheses when nothing is passed to the constructor.

for consistency